src/__tests__/index.spec.js

```diff
@@ -67,3 +67,10 @@ test('ignore non plain objects', () => {
 	});
 	expect(result).toMatchSnapshot();
 });
+
+test('load absolute path on windows too', () => {
+	const result = deabsDeep({
+		winPath: __dirname,
```

Comment:
How does this test test Windows specific behavior if we run tests on unix?

Reply:
it does not... I just ran it on windows. 
I will change it.